<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenVPNProp</class>
 <widget class="QWidget" name="OpenVPNProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>422</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QPushButton" name="btnAdvanced">
     <property name="text">
      <string>Advanced...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkShowPasswords">
     <property name="text">
      <string>Show Passwords</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel3">
       <property name="text">
        <string>Gateway:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>gateway</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="gateway"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelConnectionType">
       <property name="text">
        <string>Connection type:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>cmbConnectionType</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KComboBox" name="cmbConnectionType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>X.509 Certificates</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pre-shared Key</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Password</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>X.509 With Password</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel2">
           <property name="text">
            <string>CA file:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>x509CaFile</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KUrlRequester" name="x509CaFile"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel3_2">
           <property name="text">
            <string>Certificate:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>x509Cert</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KUrlRequester" name="x509Cert"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel4">
           <property name="text">
            <string>Key:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>x509Key</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="KUrlRequester" name="x509Key"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Key password:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="KLineEdit" name="x509KeyPassword">
             <property name="passwordMode">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KComboBox" name="x509KeyPasswordStorage">
             <item>
              <property name="text">
               <string>Store</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Always Ask</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Not Required</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>347</width>
           <height>206</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel5">
           <property name="text">
            <string>Shared Key:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>pskSharedKey</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KUrlRequester" name="pskSharedKey"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel6">
           <property name="text">
            <string>Local IP:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>pskLocalIp</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="KLineEdit" name="pskLocalIp"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="textLabel7">
           <property name="text">
            <string>Remote IP:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>pskRemoteIp</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="KLineEdit" name="pskRemoteIp">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Key Direction:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KComboBox" name="cmbKeyDirection">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="whatsThis">
            <string>If key direction is used, it must be the opposite of that used on the VPN peer.</string>
           </property>
           <item>
            <property name="text">
             <string comment="like in None setting selected">None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel8">
           <property name="text">
            <string>CA File:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>passCaFile</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KUrlRequester" name="passCaFile"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel9">
           <property name="text">
            <string comment="like in Username for athentication">Username:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>passUserName</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KLineEdit" name="passUserName"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Password:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="KLineEdit" name="passPassword">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="passwordMode">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KComboBox" name="passPasswordStorage">
             <item>
              <property name="text">
               <string>Store</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Always Ask</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Not Required</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QFormLayout" name="formLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel2_2">
           <property name="text">
            <string>CA file:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>x509PassCaFile</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KUrlRequester" name="x509PassCaFile"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel3_2_2">
           <property name="text">
            <string>Certificate:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>x509PassCert</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KUrlRequester" name="x509PassCert"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel4_2">
           <property name="text">
            <string>Key:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>x509PassKey</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="KUrlRequester" name="x509PassKey"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Key password:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="textLabel9_2">
           <property name="text">
            <string comment="like in Username for athentication">Username:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>x509PassUsername</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="KLineEdit" name="x509PassUsername"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="KLineEdit" name="x509PassKeyPassword">
             <property name="passwordMode">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KComboBox" name="x509PassKeyPasswordStorage">
             <item>
              <property name="text">
               <string>Store</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Always Ask</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Not Required</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="KLineEdit" name="x509PassPassword">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="passwordMode">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KComboBox" name="x509PassPasswordStorage">
             <item>
              <property name="text">
               <string>Store</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Always Ask</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Not Required</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>gateway</tabstop>
  <tabstop>cmbConnectionType</tabstop>
  <tabstop>x509CaFile</tabstop>
  <tabstop>x509Cert</tabstop>
  <tabstop>x509Key</tabstop>
  <tabstop>x509KeyPassword</tabstop>
  <tabstop>x509KeyPasswordStorage</tabstop>
  <tabstop>chkShowPasswords</tabstop>
  <tabstop>pskSharedKey</tabstop>
  <tabstop>cmbKeyDirection</tabstop>
  <tabstop>pskLocalIp</tabstop>
  <tabstop>pskRemoteIp</tabstop>
  <tabstop>passCaFile</tabstop>
  <tabstop>passUserName</tabstop>
  <tabstop>passPassword</tabstop>
  <tabstop>passPasswordStorage</tabstop>
  <tabstop>x509PassCaFile</tabstop>
  <tabstop>x509PassCert</tabstop>
  <tabstop>x509PassKey</tabstop>
  <tabstop>x509PassKeyPassword</tabstop>
  <tabstop>x509PassKeyPasswordStorage</tabstop>
  <tabstop>x509PassUsername</tabstop>
  <tabstop>x509PassPassword</tabstop>
  <tabstop>x509PassPasswordStorage</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cmbConnectionType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
